src/dotenv_vault/main.py

```diff
@@ -11,6 +11,9 @@
 import dotenv.main as dotenv
 
 
+DOTENV_VAULT_PATH = dotenv.find_dotenv(".env.vault")
```

Comment:
change this to instead find the `.env` file and then find the `.env.vault` file right next to it.

That would then match user's expected behavior more closely. This is what the node lib does.

https://github.com/motdotla/dotenv/blob/master/lib/main.js#L152

Reply:
@motdotla , made a new change. It should look for .env first, and expect .env.vault to be in same directory. Else, it will scream at you

Reply:
<img width="1123" alt="image" src="https://github.com/dotenv-org/python-dotenv-vault/assets/5308282/62448969-3d04-4d1f-a38a-ae291352ba38">